src/assets/colors.js

```diff
@@ -2,6 +2,7 @@
 
 export const BLUE = '#2196F3'
 export const GRAY = '#9E9E9E'
+export const GRAY_900 = '#212121'
```

Comment:
this is a nice way to name colors moving forward. It's very explicit and I know exactly which color you are referring to. I'm inclined to changing all color names in my next PR to this kind of format

Reply:
yep, just matching what's on https://www.materialui.co/colors!